corehq/apps/cleanup/management/commands/rebuild_cases_with_mismatched_forms.py

```diff
@@ -14,7 +14,10 @@ def wrapper(row):
         id = row["id"]
         doc = ExceptionRecord.get(id)
         domain = doc.domain if hasattr(doc, "domain") else ""
-        domain, case_id = domain.split(',')
+        try:
+            domain, case_id = domain.split(',')
+        except ValueError:
+            return {'exception': doc}
```

Comment:
ha! It is an ExceptionRecord doc, I thought it is some old domain doc. Could change it to `exception_doc`. But this is fine too.